carapp/db_models.py

```diff
@@ -27,20 +28,24 @@ class Car(db.Model):
     country_now = db.Column(db.String(20), nullable=True)
     city = db.Column(db.String(20), nullable=True)
     mileage = db.Column(db.String(30), nullable=True)
-    technical_condition = db.Column(db.String(180), nullable=True)
+    technical_condition = db.Column(db.Text, nullable=True)
     gear_box = db.Column(db.String(20), nullable=True)
     fuel_type = db.Column(db.String(20), nullable=True)
-    engine = db.Column(db.String(20), nullable=True)
+    engine = db.Column(db.String(50), nullable=True)
     color = db.Column(db.String(20), nullable=True)
-    safety = db.Column(db.String(180), nullable=True)
+    safety = db.Column(db.Text, nullable=True)
     wanted = db.Column(db.String(10), nullable=True)
-    multimedia = db.Column(db.String(20), nullable=True)
-    comfort = db.Column(db.String(200), nullable=True)
+    multimedia = db.Column(db.Text, nullable=True)
+    comfort = db.Column(db.Text, nullable=True)
     picture = db.Column(db.String(20), nullable=True)
+    price = db.Column(db.String(20), nullable=True)
+    drivetrain = db.Column(db.String(20), nullable=True)
+    warranty = db.Column(db.String(20), nullable=True)
+    fuel_consumption = db.Column(db.String(80), nullable=True)
     date_added = db.Column(
         db.DateTime(), nullable=False, default=datetime.datetime.utcnow
     )
     user_id = db.Column(db.Integer, db.ForeignKey("user.id"), nullable=False)
 
     def __repr__(self):
-        return f"Car({self.title}, {self.picture}, {self.date_added})"
+        return f"Car({self.title}, {self.picture}, {self.price}, {self.user_id})"
```

Comment:
Great, I see evidence of debugging, haha!

Reply:
Haha :)